py_zipkin/logging_helper.py

```diff
@@ -97,6 +101,7 @@ def log_spans(self):
                 )
 
             # Collect, annotate, and log client spans from the logging handler
+            # In case of root client span client_spans list is empty
```

Comment:
I removed my comment. May be it would be better to rename client_spans (self.log_handler.client_spans) to child_spans?